centos_ci.sh

```diff
@@ -292,14 +292,34 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  REPO_OWNER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  git pull -f $REPO $BRANCH # Get changes from remote repo/branch
```

Comment:
No branch creation? Not needed? Did you test the 'pull -f' against a repo/branch which would not be a fast forward of the master?